### ISSUE TITLE
Handlebars each loops error

### ISSUE THREAD

**Issue by marcinsydor** [dart-bridge/framework#16]
I'm testing handlebars templates on server side. I have a loop:

``````
```{{#each models}}
    <article>
        <h1>row test</h1>
    </article>
```{{/each}}
``````

And getting error where I'm reloading projects.

```
'file:///Users/user/Repos/lab/bridge/.templates.dart': error: line 39 pos 36: semicolon expected
    ${await () async {var c = each models;compile() async => """
```

**Comment by marcinsydor** [comment]
There is also an error if I'm trying to load handlebars partials. `{{>partials.header}}`

```
in ShutdownIsolate: 'file:///Users/marcinsydor/Repos/lab/bridge/.templates.dart': error: line 39 pos 7: unexpected token '>'
    ${>partials.header}
      ^
```

**Comment by emilniklas** [comment]
Partials aren't implemented, but they should be an easy addition. I'm gonna have to look at that handlebars loop.

**Comment by emilniklas** [comment]
Fixed in `1.0.0-alpha3`. See closing comment in #14
